docs/rfcs/managed.md

```diff
@@ -0,0 +1,142 @@
+# TITLE
+
+**Status:** *Draft*
+
+**Author:** Antoine Toulme (@atoulme)
+
+**Date:** 2025-03-20
+
+## Objective
+
+Condense all best practices and deployment patterns of OpenTelemetry components into a single custom resource
+that covers the feature set of the OpenTelemetry Helm chart.
+
+## Summary
+This request for comment aims to establish a new approach to management of resources in Kubernetes clusters,
+specifically in Openshift environments, with the explicit goal of taking over the whole cluster, and making decisions
```

Comment:
Why specifically OpenShift?

Reply:
that is the population we specifically target here. Openshift has heightened security requirements and offers operators as first class experience. For users using the vanilla Kubernetes experience, the Helm chart might be a better fit.

Reply:
I would say that was not the philosophy we tried to follow for the operator. We created some OpenShift specific features but just because vanilla Kubernetes could not do those specific stuff. In this case, I don't see why the non-OpenShift environments should be excluded.